.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,13 +1,7 @@
 <!--
-....................................
+Thank you for submitting to the TechEmpower Framework Benchmarks!
 
-MAKE SURE YOU ARE OPENING A PULL
-REQUEST AGAINST THE CORRECT BRANCH
+If you are submitting a new framework please make sure that you add the appropriate line in the `.travis.yml` file for proper integration testing and that an appropriate `README.md` is added in your framework directory with information about the framework and a link to it's homepage and documentation.
```

Comment:
Just for consistency, there should be a comma between "framework" and "please"
Also, it should be "and a link to its homepage" not "it's homepage"